en/reference/services-content.html

```diff
@@ -910,6 +918,24 @@ <h2 id="initialize">initialize</h2>
 &lt;/initialize&gt;
 </pre>
 
+<h2 id="lidspace">lidspace</h2>
+<p>
+Contained in <a href="#searchnode">searchnode</a>, optional.
+Tune settings related to how lidspace is managed. Optional sub-elements:
+</p>
+<ul>
+  <li id="lidspace-max-bloat-factor"><code>max-bloat-factor</code>:
+    Maximum bloat allowed before lidspace compaction is started. Compaction is moving a document
+    from a high lid to a lower lid. Cost is similar to feeding a document and removing it.
+    Default value is 0.01 or 1% of total lidspace. Will be increased to target of 0.50 or 50%.
```

Comment:
I did not understand "Will be increased to target of 0.50 or 50%."

Reply:
Default is today 0.01 or 1%. This is a very bad default, and will soon, at some point in time be adjusted to 0.50, or 50%.

Reply:
Forgot to reload ?
I see it.